diff --git a/src/controllers/Flare-Controller.coffee b/src/controllers/Flare-Controller.coffee
index 491f635..9ece9bf 100644
--- a/src/controllers/Flare-Controller.coffee
+++ b/src/controllers/Flare-Controller.coffee
@@ -62,6 +62,11 @@ class Flare_Controller
       @.page_MainPage_no_user      = '/flare/index'
       @.loginUser()
 
+  send_Angular_Coffee: (req,res)=>
+
+    file_Path = '../TM_Flare/angular'.path_Combine req.params.file
+    res.sendFile file_Path.real_Path()
+
   routes: ()=>
     using new Router(), ->
       flare_Controller = new Flare_Controller()
@@ -74,6 +79,7 @@ class Flare_Controller
       @.get  '/navigate/:queryId/:filters', flare_Controller.show_Navigate
       @.get  '/:page'                     , flare_Controller.render_Page
       @.get  '/'                          , (req, res)-> res.redirect '/flare/index'
+      @.get '/angular/:file' , flare_Controller.send_Angular_Coffee
       @
 
 
